jujugui/static/gui/src/app/components/entity-content-config-option/entity-content-config-option.js

```diff
@@ -39,21 +45,21 @@ YUI.add('entity-content-config-option', function() {
     },
 
     render: function() {
-      var name = this.props.name;
+      var option = this.props.option || {};
```

Comment:
Data safety. There were some tests that were rendering with an undefined option, so this was necessary to get the tests to pass. I think it may have something to do with React's lifecycle, when it first mounts the component, but nothing (state and props-wise) is set yet?